x-pack/plugins/beats_management/public/components/inputs/multi_input.tsx

```diff
@@ -48,10 +48,10 @@ class MultiFieldText extends Component<
   }
 
   public handleChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
-    const { value } = e.currentTarget;
+    const value = e.currentTarget.value.split('\n');
```

Comment:
It may be better to assign this a pluralized name since it's now going to be an array, and we aren't destructuring `value` from the event object.

Reply:
value is though sometimes a single value, its just also sometimes an array.

Reply:
won't calling `split` cause `value` to always be an array?